Cargo.toml

```diff
@@ -105,3 +106,7 @@ lto = true
 [[example]]
 name = "flash_with_rtic"
 required-features = ["stm32g474"]
+
+[[example]]
+name = "cordic"
+required-features = ["defmt"]
```

Comment:
should not be needed after #134. 

```suggestion
```

(Apparently the macro was only used and tested without formatting args.)

Reply:
This command fails now:

```sh
cargo build --examples --features stm32g431,log-rtt
```

Looks like the `log-rtt` feature no longer enables the `defmt` feature?